docs/testbed/README.testbed.BgpPeersNumberScaling.md

```diff
@@ -0,0 +1,109 @@
+# Background and Motivation
+
+When deploy a testbed with a great number of virtual ceos neighbors, we will create ceos containers on same server, however, the server doesn't have infinite resources such as memory to deploy that.
+
+To leverage the servers instead of single server, we proposal this design for deploying testbed with multiple servers.
+
+# Table of Contents
+  - [File Content Schema Change](#File-Content-Schema-Change)
+    - [testbed.yml](#testbedyml)
+    - [topo_*.yml](#topo_yml)
+  - [Code Logic Change](#Code-Logic-Change)
+    - [Parameters Constrain Validation](#Parameters-Constrain-Validation)
+    - [Create 802.1Q VLAN](#Create-8021Q-VLAN)
+    - [Create Net and Ceos Containers](#Create-Net-and-Ceos-Containers)
+    - [Create PTF Containers](#Create-PTF-Containers)
+    - [Network Connection](#Network-Connection)
+    - [Automation Script](#Automation-Script)
+    - [Changes To Be Determined](#Changes-To-Be-Determined)
+
```

Comment:
It will be better to add a high level graph here to the idea of this design and how each components interact with each other.

Reply:
The relationship of Mgmt-Test, Server/PTF/Neighbor and DUT in the new design.